pkg/ipamd/ipamd.go

```diff
@@ -385,8 +386,11 @@ func (c *IPAMContext) nodeInit() error {
 
 	metadataResult, err := c.awsClient.DescribeAllENIs()
 	if err != nil {
+		// Check error and broadcast event in case of missing EC2 permissions
+		c.CheckEC2Permissions(err)
```

Comment:
The functions returns an error, but we ignoring it here.

Reply:
Removed the return and printed an error message instead, thanks!